code/modules/species/shadekin/crew_shadekin_abilities.dm

```diff
@@ -0,0 +1,132 @@
+///  REGENERATE OTHER  ///
+//////////////////////////
+/datum/power/crew_shadekin/crewkin_regenerate_other
+	name = "Regenerate Other (50)"
+	desc = "Spend energy to heal physical wounds in another creature."
+	verbpath = /mob/living/carbon/human/proc/crewkin_regenerate_other
+	ability_icon_state = "tech_biomedaura"
+
+/mob/living/carbon/human/proc/crewkin_regenerate_other()
+	set name = "Regenerate Other (50)"
+	set desc = "Spend energy to heal physical wounds in another creature."
+	set category = "Shadekin"
+
+	var/ability_cost = 50
+
+	if(species.get_species_id() != SPECIES_ID_SHADEKIN)
+		to_chat(src, "<span class='warning'>Only a black-eyed shadekin can use that!</span>")
+		return FALSE
+	else if(stat)
+		to_chat(src, "<span class='warning'>Can't use that ability in your state!</span>")
+		return FALSE
+	else if(shadekin_get_energy() < ability_cost)
+		to_chat(src, "<span class='warning'>Not enough energy for that ability!</span>")
+		return FALSE
+
+	var/list/targets = list()
+	for(var/mob/living/L in view(1))
+		targets += L
+	if(!targets.len)
+		to_chat(src,"<span class='warning'>Nobody nearby to mend!</span>")
+		return FALSE
+
+	var/mob/living/target = input(src,"Pick someone to mend:","Mend Other") as null|anything in targets
+	if(!target)
+		return FALSE
+
+	target.add_modifier(/datum/modifier/crew_shadekin/heal_boop,1 MINUTE)
+	playsound(src, 'sound/effects/EMPulse.ogg', 75, 1)
+	shadekin_adjust_energy(-ability_cost)
+	visible_message("<span class='notice'>\The [src] gently places a hand on \the [target]...</span>")
+	face_atom(target)
+	return TRUE
+
+/datum/modifier/crew_shadekin/heal_boop
+	name = "Shadekin Regen"
+	desc = "You feel serene and well rested."
+	mob_overlay_state = "green_sparkles"
+
+	on_created_text = "<span class='notice'>Sparkles begin to appear around you, and all your ills seem to fade away.</span>"
+	on_expired_text = "<span class='notice'>The sparkles have faded, although you feel much healthier than before.</span>"
+	stacks = MODIFIER_STACK_EXTEND
+
+/datum/modifier/crew_shadekin/heal_boop/tick()
+	if(!holder.getBruteLoss() && !holder.getFireLoss() && !holder.getToxLoss() && !holder.getOxyLoss() && !holder.getCloneLoss()) // No point existing if the spell can't heal.
+		expire()
+		return
+	holder.adjustBruteLoss(-1)
+	holder.adjustFireLoss(-1)
+	holder.adjustToxLoss(-1)
+	holder.adjustOxyLoss(-1)
+	holder.adjustCloneLoss(-1)
+
+
+//////////////////////
+///  CREATE SHADE  ///
+//////////////////////
+/datum/power/crew_shadekin/crewkin_create_shade
+	name = "Create Shade (25)"
+	desc = "Create a field of darkness that follows you."
+	verbpath = /mob/living/carbon/human/proc/crewkin_create_shade
+	ability_icon_state = "tech_dispelold"
+
+/mob/living/carbon/human/proc/crewkin_create_shade()
+	set name = "Create Shade (25)"
+	set desc = "Create a field of darkness that follows you."
+	set category = "Shadekin"
+
+	var/ability_cost = 25
+
+	if(species.get_species_id() != SPECIES_ID_SHADEKIN)
```

Comment:
same as above, shouldnt this be
SPECIES_ID_SHADEKIN_BLACK_EYED
?

Reply:
Same as above